TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ultimategoal/modules/OdometryModule.java

```diff
@@ -34,7 +40,10 @@
     public double rightPodNewPosition;
     public double mecanumPodNewPosition;
 
+    private long startTime = SystemClock.currentThreadTimeMillis();
```

Comment:
elapsedRealTime is what we used. Sorry, couldn't recall for a long time.

Reply:
oh; elapsedrealtime is only since the start, so i think i'll use the calendar time. then we can like sort the files more easily